about.html

```diff
@@ -7,58 +7,73 @@
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
-
+  <link rel="stylesheet" href="css/about.css">
+  <h1>
+    <nav class="top-nav">
+        <a class="logo"><img src="lambda-black.png" alt="lambda logo"></a>
```

Comment:
Great implementation of the header image into the navbar, though not part of MVP it is only missing the href attribute.